arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/gas/projector/Call.java

```diff
@@ -116,14 +116,14 @@ public long rawStipend() {
       return 0L;
     } else {
       final long remaining = frame.getRemainingGas() - cost;
-      final long weird = remaining - remaining / 64;
+      final long sixtyThreeSixtyFourths = remaining - remaining / 64;
 
```

Comment:
what is the purpose of this? do we have a self-explanatory name for it?

Reply:
`weird` isn't a good name, is all. And what we are computing here is the `63/64-ths` of the `remaingGas` minus the `upfrontGasCost`.

Reply:
We should, if anything, rename `cost` to `upfrontGasCost`. Will do.